### PR TITLE
Split into a couple of artifacts VERTXLIB-5

### PR DESCRIPTION
**Pull request by adamdickmeiss**
Disable Sonarqube for now as, it fails when main artifact changes (or something).

### CODE REVIEW

File: pg-testing/src/main/java/org/folio/tlib/postgres/TenantPgPoolContainer.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package org.folio.tlib.postgres;
```
Comment (julianladisch) [inline pg-testing/src/main/java/org/folio/tlib/postgres/TenantPgPoolContainer.java:1]: This package is already used in vertx-lib.
Can you add a module-info.java for each maven submodule and fix all issues to achieve [Java 9+ modularity](https://developer.ibm.com/tutorials/java-modularity-5/)?

Comment (adamdickmeiss) [reply pg-testing/src/main/java/org/folio/tlib/postgres/TenantPgPoolContainer.java:1]: Two hours and a couple of stackoverflow lookups.. No luck.. WIth a module-info.java with a few requires doesn't cut it as everything this depend on , vertx, okapi, cql-java are all unnamed. This library would be the first in folio to adopt module-info.java and one of the very few on GitHub to do so.

Comment (adamdickmeiss) [reply pg-testing/src/main/java/org/folio/tlib/postgres/TenantPgPoolContainer.java:1]: OK. I have at least some `requires` that works now, so I this should be done soon.

Comment (julianladisch) [reply pg-testing/src/main/java/org/folio/tlib/postgres/TenantPgPoolContainer.java:1]: The first step is automatic modules: https://github.com/folio-org/folio-vertx-lib/pull/18

That pull request can be merged into this pull request.

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

**Pull request merged**